app/controllers/employment/AmountOfTipsOnP60Controller.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.employment
+
+import audit.AuditService
+import config.{AppConfig, ErrorHandler}
+import controllers.employment.routes.{CheckEmploymentDetailsController, OtherPaymentsController}
+import controllers.predicates.{AuthorisedAction, InYearAction}
+import forms.AmountForm
+import play.api.data.Form
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.EmploymentSessionService
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import utils.SessionHelper
+import views.html.employment.{AmountOfTipsOnP60View, CheckEmploymentDetailsView}
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class AmountOfTipsOnP60Controller @Inject()(implicit val cc: MessagesControllerComponents,
```

Comment:
should the controller name be tip payments that are not on P60? or maybe EmploymentOtherPaymentsController?

Reply:
updated to be OtherPaymentsAmount view/controller